src/components/Svgs/PaginationArrow.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { PseudoBox, IconButton } from '@chakra-ui/core';
+import PropTypes from 'prop-types';
+
+/**
+ * @function PaginationArrow
+ *
+ * @param {function} onClick - Callback function that runs when icon is clicked
+ * @param {boolean} isDisabled - Boolean that determines whether the icon is disabled
+ * @param {string} icon - Name of the svg icon from the theme.js file
+ *
+ */
+
+const PaginationArrow = ({ onClick, isDisabled, icon }) => {
+  return (
+    <PseudoBox
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      height={10}
+      width={10}
+      stroke="rbb-black"
+      _hover={{
+        color: 'rbb-orange',
+      }}
+      _active={{
+        color: 'white',
+      }}
+      _focus={{
+        color: 'white',
+      }}
+    >
+      <IconButton
+        onClick={onClick}
+        aria-label="Next"
+        aria-labelledby={icon}
+        variant="unstyled"
+        fill="white"
```

Comment:
The `fill` needs to be based on the current state of the button:

- default: `#001514`
- active/focus: `#ffffff`
- hover: `#F46036`

I'm not 100% sure how to do that once you migrate over to actual SVGs though.

Reply:
The only white value in the theme is #F7F7F2. Should I inline #ffffff, create a new white value in the config, or use #F7F7F2?